skbio/metadata/_interval.py

```diff
@@ -0,0 +1,699 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import operator
+from functools import wraps
+
+from ._intersection import IntervalTree
+from skbio.util._decorator import experimental
+
+
+class Interval:
+    """Stores the location and metadata of an interval feature.
```

Comment:
"interval feature" isn't super clear to me. Can you update "interval features" references to "a feature over intervals"?